packages/elements/src/components/API/APIWithStackedLayout.tsx

```diff
@@ -141,27 +137,27 @@ const Item = React.memo<{ item: OperationNode }>(({ item }) => {
         {hideTryIt ? (
           <Box as={ParsedDocs} node={item} headless p={4} />
         ) : (
-          <Tabs
-            className="PreviewTabs mx-auto"
-            selectedTabId={tabId}
-            onChange={(tabId: PanelTabId) => setTabId(tabId)}
-            renderActiveTabPanelOnly
-          >
-            <Tab
-              id="docs"
-              title="Docs"
-              className="p-4"
-              panel={<ParsedDocs node={item} headless location={location} />}
-            />
-            <Tab
-              id="tryit"
-              title="Try It"
-              className="p-4"
-              panel={<TryItWithRequestSamples httpOperation={item.data} />}
-            />
+          <Tabs appearance="line">
+            <TabList>
+              <Tab>Docs</Tab>
+              <Tab>TryIt</Tab>
+            </TabList>
+
+            <TabPanels>
+              <TabPanel>
+                <ParsedDocs node={item} headless location={location} />
+              </TabPanel>
+              <TabPanel>
+                <TryItWithRequestSamples httpOperation={item.data} />
+              </TabPanel>
+            </TabPanels>
           </Tabs>
         )}
       </Collapse>
     </div>
   );
 });
+
+const Collapse: React.FC<{ isOpen: boolean }> = ({ isOpen, children }) => {
+  return <Box style={{ display: isOpen ? 'block' : 'none' }}>{children}</Box>;
```

Comment:
If you decided to add a `Box` you could might as well just use it's `display={}` prop, couldn't you?